app/controllers/impersonations_controller.rb

```diff
@@ -12,4 +13,10 @@ def destroy
     stop_impersonating_user
     redirect_to users_path
   end
+
+  private
+
+  def clear_session_organization_id
+    session[:organization_id] = nil
+  end
```

Comment:
Should we need to set this to something other than `nil` given that there should be a default organization we can use?

Reply:
Once we clear the session, rails will redirect, hit the application controller again and set a new org if that makes sense.

Reply:
Makes sense but when an impersonation ends, we aren't wiping the entire session are we?

Reply:
No just the organization id.